packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@backstage/eslint-plugin",
+  "description": "Backstage ESLint plugin",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://backstage.io",
```

Comment:
no dependencies, like on `@manypkg/get-packages`?

Reply:
Derp, yep. Added some lint rules to catch more of that in the future.